### ISSUE TITLE
Feature request - Simple Repeater Field

### ISSUE THREAD

**Issue by leurdo** [htmlburger/carbon-fields#538]
Hello,
It would be great to have a simple repeater field, just a list of one repeated field. Like this:
![image](https://user-images.githubusercontent.com/16862217/40734041-f98ace1a-643f-11e8-953d-f374ef25eacf.png)
Thank you!

**Comment by KevinSupertramp** [comment]
I agree that would be awesome. I might have a look into it if I can get any free time!

**Comment by agengdp** [comment]
+1
